index.d.ts

```diff
@@ -8,10 +8,14 @@ declare module 'async-sema' {
     drain(): Promise<string[]>;
     
     nrWaiting(): number;
+
+    v(): Promise<string>;
+
+    acquire(token?: string): void;
```

Comment:
This seems incorrect @OlliV 
should it not be `acquire(): Promise<string>`?

Reply:
That's correct, thanks.